src/ui_fsmenu/launch_game.cc

```diff
@@ -75,6 +75,20 @@ LaunchGame::LaunchGame(MenuCapsule& fsmm,
                              UI::DropdownType::kTextual,
                              UI::PanelStyle::kFsMenu,
                              UI::ButtonStyle::kFsMenuMenu),
+     win_condition_duration_(&right_column_content_box_,
+                             0,
+                             0,
+                             200,
+                             200,
+                             Widelands::kDefaultWinConditionDuration,
+                             15,        // 15 minutes minimum gametime
+                             512 * 60,  // 512 hours maximum gametime (arbitrary limit)
+                             UI::PanelStyle::kFsMenu,
+                             "",
```

Comment:
I miss the label. Does it really not fit?

Reply:
When the string expands to "x hours and y minutes" it is impossible to fit a label here:
![grafik](https://user-images.githubusercontent.com/48293446/198847744-f15e6428-78dd-4de5-8854-21ee3b11e7f9.png)

Reply:
What is your resolution/window size? I've noticed that resizing of this window has a bug, I've even created screenshots for a bug report, just haven't reported it yet. So I wonder how much width we could fit in at lower resolutions.

Reply:
This is at 1024×768. At 1920×1080 fullscreen it fits fine.

Reply:
I've checked the launch game windows. In single player it should be split at half width anyway, because there's much less info on the player list side. This way it should be possible to make the width of the game settings part at least 360px even at 800x600. (perhaps some paddings need to be decreased too to achieve this). For multiplayer it may be a little less though, because there are 3 columns (clients, player slots, and settings).

Still some more width can be spared if the width of the big step buttons is decreased by decreasing the big padding beside the double arrows.